autoswitch_virtualenv.plugin.zsh

```diff
@@ -142,7 +142,7 @@ function _check_path()
 function _activate_poetry() {
     # check if any environments exist before trying to activate
     # if env list is empty, then no environment exists that can be activated
-    local name="$(poetry env list --full-path | cut -d' ' -f1)"
+    local name="$(poetry env list --full-path | sort -k 2 | tail -n 1 | cut -d' ' -f1)"
```

Comment:
I'm curious why you chose to use sort here? Is it so that you can detect the activated venv?

Reply:
Yeah, the sort is so we prefer the activated venv. I'm not sure if it's possible to not have an activated venv; if we're guaranteed to have one we could obviously just use grep.